unittests/AST/ASTImporterTest.cpp

```diff
@@ -112,11 +112,15 @@ struct Fixture : ::testing::Test {
   std::unique_ptr<ASTUnit> FromAST, ToAST;
   const char *const InputFileName = "input.cc";
   const char *const OutputFileName = "output.cc";
+  std::string FromCode, ToCode, Code; //Buffers, must live in test scope
 
   std::tuple<Decl *, Decl *>
-  getImportedDecl(const std::string &FromCode, Language FromLang,
-                  const std::string &ToCode, Language ToLang,
```

Comment:
LLVM do not use postfix underscores fo parameter names, I think we should stick to LLVM guidelines.

Reply:
Meanwhile I just found the coding conventions:
https://llvm.org/docs/CodingStandards.html
I have to admit I've never read it before, time to fix this :)